01-module-system/helpers.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs')
+const clr = require ("chalk")
+
+const getNotes = () => {
+    try{
+const data = fs.readFileSync('notes.json')
+const content = data.toString()
+const jsonData=JSON.parse(content)
+return jsonData 
```

Comment:
pleases fix indentation

Reply:
saving to jsonData is redundant please return parsed content without saving into variable